default.nix

```diff
@@ -140,7 +140,8 @@ with nixpkgs; naersk.buildPackage rec {
   postConfigure = ''
       ${nixpkgs.lib.optionalString providedCargoHome
          ''
-           export CARGO_HOME=${cargoHome}
+           cp -r ${cargoHome} .cargo_home
```

Comment:
Somehow `cargo` attempts to write inside of the `CARGO_HOME` folder, which previously was stored inside of the `/nix` folder and so it was read-only. This copies it into the build folder.

Reply:
Also add this information to the comment above.